native/python/app/keystore/generate-certs.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Usage: ./generate-certs.sh [zosmfProfile] [certName] [serverCert] [caCerts...]
+set -e
+
+function zowe() {
+    npx -y zowe "$@" --zosmf-p $zosmfProfile
+}
+
+zosmfProfile=${1:-$(npx -y zowe config list defaults | grep "zosmf:" | awk '{print $NF}')}
+certName=${2:-$zosmfProfile}
+certLabel=${3:-ROOTSTAR}
+caLabels=("${@:4}")
+if [ "$#" -lt 4 ]; then
+  caLabels=("DigiCert CA" "DigiCert Global Root CA")
+fi
+datasetHlq=$(ZOWE_SHOW_SECURE_ARGS=true zowe zosmf check status --show-inputs-only | grep "user:" | awk '{print $NF}')
+keyPassword=password
+opensslBin=${OPENSSL_BIN:-openssl}
+
+# Export server cert and CA certs to data sets
+zowe tso issue cmd "RACDCERT SITE EXPORT(LABEL('$certLabel')) DSN('$datasetHlq.APIML.CERT.P12') FORMAT(PKCS12DER) PASSWORD('$keyPassword')"
```

Comment:
Should we do a test for the ESM being used?
I kind of forgot that the script won't work as-is (on another system I'm trying to test with) since it's RACF dependent 😋